### DISCUSSION TITLE
1.3.4 Release

### DISCUSSION THREAD

**Discussion by jertel** [jertel/vuegraf#74] [General]
- Upgrade to PyEmVue 0.15.0 to resolve API change by Emporia

Note that this build's Docker Image is much larger than the previous release. This is due to the need for the newer cryptography Python package's Rust code to be compiled on Alpine distros since there is no pre-built package available on Alpine. Additionally, the `docker build` command will take significantly longer than it used to, for those that prefer to build their own images.

<hr /><em>This discussion was created from the release <a href='https://github.com/jertel/vuegraf/releases/tag/1.3.4'>1.3.4 Release</a>.</em>

**Comment by dwduback** [comment]
My vuegraf broke on 10/13/2021. I assumed it was because Emporia changed their API as suggested on github. I did not push anything to the Emporia nor did I knowingly alter anything on the Docker stuff in the vuegraf stack.

I see the reference to armV7 being disabled due to an issue with Rust.

At present, I only use the pi4 to run docker with the vuegraf stack and portainer. A pizero is also writing to the influxdb container which contains the vue data up to 10/13/2021 and that is going OK...

Python 3.7.3
I also see that pip3 reports the following on my pi4(4g,:


influxdb==5.3.1
influxdb-client==1.20.0
pycognito==2021.3.1
pyemvue==0.15.0
requests==2.26.0
requests-oauthlib==1.0.0
 

   Static hostname: piFOUR
         Icon name: computer
  Operating System: Raspbian GNU/Linux 10 (buster)
            Kernel: Linux 5.10.52-v7l+
      Architecture: arm

 So to me it is conceivable to install vuegraf outside of Docker (with a LOT more knowledge than I have).

My eyes glaze over when trying to understand Docker containers. Are there plans to include armV7  again or should I move the stack to an x86 machine?

**Reply by jertel** [reply]
I've re-enabled armv7. There is now a `latest` tag of the image at [hub.docker.com](https://hub.docker.com/r/jertel/vuegraf/tags?page=1&name=latest).

**Reply by dwduback** [reply]
Many thanks for publicizing this container and fixing the latest challenge. Some work to do to import saved data but this is very doable...
